### ISSUE TITLE
Is this repository still needed?

### ISSUE THREAD

**Issue by ketiltrout** [chime-experiment/ch_util-stub#1]
`ch_util` is public now.  Is there anyone still using this repository?  If possible we should migrate away from this.

**Comment by anjakefala** [comment]
No! Not needed. I did change some actions from `stub` to `ch_util`, but maybe did not catch them all? You could delete the repo, and see which actions break. =)
